test/v2/controllers/V2ArrivalsControllerSpec.scala

```diff
@@ -675,4 +682,80 @@ class V2ArrivalsControllerSpec
     }
   }
 
+  "GET  /traders/:EORI/movements/arrivals" - {
```

Comment:
endpoint looks incorrect should be GET  /traders/movements/arrivals? for controller